wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -26,12 +26,13 @@ <h1 class="icon icon-doc-empty-inverse">
         <footer role="contentinfo">
             <nav aria-label="{% trans 'Actions' %}">
                 <ul>
-                    <li class="actions">
-                        <div class="dropdown dropup dropdown-button match-width">
-                            <button type="submit" class="button action-save button-longrunning" data-clicked-text="{% trans 'Saving…' %}"><span class="icon icon-spinner"></span><em>{% trans 'Save draft' %}</em></button>
-                            {{ action_menu.render_html }}
-                        </div>
-                    </li>
+                  <li class="actions">
+                      <div class="dropdown dropup dropdown-button match-width {% if is_revision %}warning{% endif %}">
+                          {{ action_menu.render_default_html }}
```

Comment:
Looks like this has been moved into `render_html` so we should delete this line

Reply:
Removed on https://github.com/gasman/wagtail/commits/rjpruitt16-SaveDraft